tsconfig.json

```diff
@@ -27,6 +27,7 @@
         "pretty": true,
         "preserveConstEnums": true,
         "removeComments": false,
+        "skipLibCheck": true, // Avoid the compile error with office-ui-fabric-react
```

Comment:
We need to enable this option until https://github.com/OfficeDev/office-ui-fabric-react/issues/382 will be fixed